server/emits.py

```diff
@@ -82,13 +82,16 @@ def gesture_handler(sid, data):
         else:
             gesture['data'] = data
 
+    model = nn.NeuralNetworkBlocksworldModel('./model.h5')
```

Comment:
Should we include the `.h5` file under version control?

Reply:
It's around 17mb. I can add it, but I don't think that it will make sense.